core/src/test/scala/unit/kafka/zk/KafkaZkClientTest.scala

```diff
@@ -276,6 +301,49 @@ class KafkaZkClientTest extends ZooKeeperTestHarness {
 
     zkClient.deleteAclChangeNotifications()
     assertTrue(zkClient.getChildren(AclChangeNotificationZNode.path).isEmpty)
+  }
+
+  @Test
+  def testDeleteTopicPathMethods() {
```

Comment:
Saw the following unit test failure.

kafka.zk.KafkaZkClientTest > testDeleteTopicPathMethods FAILED
    java.lang.AssertionError: expected:<List(topic1, topic2)> but was:<Buffer(topic2, topic1)>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at kafka.zk.KafkaZkClientTest.testDeleteTopicPathMethods(KafkaZkClientTest.scala:318)